src/components/Charts/ChartRt.tsx

```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+import moment from 'moment';
+import { isUndefined } from 'lodash';
+import { min as d3min, max as d3max } from 'd3-array';
+import { Group } from '@vx/group';
+import { ParentSize } from '@vx/responsive';
+import { scaleLinear, scaleTime } from '@vx/scale';
+import { GridRows } from '@vx/grid';
+import { curveNatural } from '@vx/curve';
+import { AxisBottom, AxisLeft } from '@vx/axis';
+import { LinePath, Area } from '@vx/shape';
+import { RectClipPath } from '@vx/clip-path';
+import { useTooltip } from '@vx/tooltip';
+import { localPoint } from '@vx/event';
+import { ProjectionDataset, RT_TRUNCATION_DAYS } from '../../models/Projection';
+import { CHART_END_DATE, CASE_GROWTH_RATE, Zones } from '../../enums/zones';
+import BoxedAnnotation from './BoxedAnnotation';
+import HoverOverlay from './HoverOverlay';
+import RectClipGroup from './RectClipGroup';
+import {
+  formatDecimal,
+  getChartRegions,
+  getTruncationDate,
+  getZoneByValue,
+  last,
+  randomizeId,
+} from './utils';
+import * as Style from './Charts.style';
+
+type PointRt = {
+  x: number;
+  y: {
+    rt: number;
+    low: number;
+    high: number;
+  };
+};
```

Comment:
i'd imagine this is coming from the projection so we should use the data type from that?

Reply:
almost ... `Column` has `{ x: number; y: any; }`, so I'm taking `RtRange` from `Projection.ts` to redefine `PointRt`

```ts
type PointRt = Omit<Column, 'y'> & {
  y: RtRange;
};
```

Not sure what the best practice is, let me know if there is a better way!

Reply:
Updated!